CraftingRevisions/BlueprintMapper.cs

```diff
@@ -140,5 +144,25 @@ private static T[] NotNull<T>(T[] array)
 
 			return result;
 		}
+		/// <summary>
+		/// was previously using GetItems()
+		/// </summary>
+		/// <param name="names"></param>
+		/// <param name="counts"></param>
+		/// <returns>Il2CppReferenceArray required by BlueprintData.m_RequiredGear</returns>
+		internal static Il2CppReferenceArray<BlueprintData.RequiredGearItem> GetRequiredGearItems(string[] names, int[] counts)
+		{
+			List<BlueprintData.RequiredGearItem> list = new();
```

Comment:
I feel like you should be able to create an `Il2CppReferenceArray` directly or with an implicit conversion from a normal array. A list is wasteful.